core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -134,11 +134,19 @@ class SocketServer(val config: KafkaConfig, val metrics: Metrics, val time: Time
   /**
    * Shutdown the socket server
    */
-  def shutdown() = {
-    info("Shutting down")
+  def stop() = {
```

Comment:
We need to update the comment and maybe we can call this `stopRequestHandling` or something? Also, it would probably be a bit nicer if `shutdown` called this if `stop` had not been invoked (would require maintaining an `AtomicBoolean` probably). If you don't think that's worth it, we should document in `shutdown` that this method must also be invoked.